src/storage/src/lib.rs

```diff
@@ -50,6 +50,7 @@
 #![feature(assert_matches)]
 #![feature(is_sorted)]
 #![feature(btree_drain_filter)]
+#![cfg_attr(coverage, feature(no_coverage))]
```

Comment:
What's this line for? 🤯

Reply:
Get it. It's for enabling the `no_coverage` feature, instead of marking the crate `no_coverage`.